research-software-directory/RSD-as-a-service
# 482 about organisation section

# Add organisation about page

Closes #482
Closes #582

Changes proposed in this pull request:
*  I implemented autosave feature and removed Save button similar to edit software and projects page
*  The markdown section is added to the settings. Organisation admin can add the about text in markdown format to enable additional menu option (about section)
*  About section is allowed at all levels, the top organisation and the research units (children)

How to test:
* `make start` to completely reset and rebuild your local rsd (to stop use `docker-compose down` when you are done)
* login as rsd_admin or maintainer of an organisation
* you will always see about section as maintainer. If there is no content, the info will be shown. You can click on bold text to open settings section
* add markdown in the settings (About page section)
* visit about section and confirm that content is shown
* logout from rsd, visit organisation and confirm that about page is shown

## Example organisation about section (public view)
![image](https://user-images.githubusercontent.com/9204081/196142069-61f1c90c-c0e0-4521-ad6f-803fe3485ec9.png)

## Example organisation (admin view - without about content)
![image](https://user-images.githubusercontent.com/9204081/196245752-c10100c2-db44-409a-b404-cc11510261d9.png)

## Example organisation settings with markdown (admin view)
![image](https://user-images.githubusercontent.com/9204081/196141756-3c727574-e387-423a-b5c0-937ef139aa27.png)

PR Checklist:
